easyovs/cli.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, stdin=sys.stdin, foreground=True):
             debug("username = %s\n" % os.getenv('OS_USERNAME') or
                   cfg.CONF.OS.username)
             passwd = os.getenv('OS_PASSWORD') or cfg.CONF.OS.password
-            passwd = passwd[:len(passwd)/4] + "****" + passwd[-len(passwd)/4:]
```

Comment:
Why comment out this line?

Reply:
> Why comment out this line?

changed it back, just my makeshift way when installed it several months ago:)